pkg/operator/k8sutil/pod.go

```diff
@@ -118,6 +118,32 @@ func GetSpecContainerImage(spec v1.PodSpec, name string, initContainer bool) (st
 	}
 	return image.Image, nil
 }
+func AddUnreachableNodeToleration(podSpec *v1.PodSpec, tolerationSeconds int64) {
+	// Node controller will wait 40 seconds by default before mark a node as
+	// unreachable (node.kubernetes.io/unreachable). This function adds a
+	// Toleration of <seconds> in the pod for this
+	// situation. After 40 + n seconds the pod will be scheduled in other node
+	// Only one <toleration> to <unreachable> nodes can be added
+	urToleration := v1.Toleration{Key: "node.kubernetes.io/unreachable",
+		Operator:          "Exists",
+		Effect:            "NoExecute",
+		TolerationSeconds: &tolerationSeconds}
+
+	var replaced bool = false
```

Comment:
A bool is always false by default so you don't need to set `= false`.

Reply:
thx!